Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,11 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.app {
+  height: 500px;
+  width: 45%;
```

Comment:
A better user experience would be to set width to 100% and restricting the width with `max-width`.  Why? When . the viewport gets really small say 300px, the card will only be about 130px wide, and the white space in the window would be greater than the card width.